xx.md

```diff
@@ -0,0 +1,135 @@
+- `relays` - a list of nostr relays
+-
+## Android Push Alerts
+
+Android push alert requests are `kind 32833` events which can be used to request Android push notifications.
+
+The following additional tags are defined:
+
+- `device_token` indicates a Firebase Cloud Messaging token
+
+The push notification SHOULD include the following JSON-encoded data:
+
+- `event` - a [truncated nostr event](#event-truncation)
+- `relays` - a list of nostr relays
+
+## iOS Push Alerts
+
+iOS push alert requests are `kind 32834` events which can be used to request iOS push notifications.
+
+The following additional tags are defined:
+
+- `device_token` indicates an APNs token
+- `bundle_identifier` indicates a iOS app bundle identifier
+
+The push notification SHOULD include the following JSON-encoded data:
+
+- `event` - a [truncated nostr event](#event-truncation)
+- `relays` - a list of nostr relays
+
+## Event Truncation
+
+Most push notification standards limit payloads to 4KB, beyond which notifications are not delivered. Since nostr events may be greater than this limit, notifiers SHOULD truncate events when sending push notifications in the following way:
+
+- `content` MAY be omitted
+- `tags` MAY be omitted
+- `sig` SHOULD be removed if either `content` or `tags` are not sent
+
+# Heartbeats
+
+Clients MAY send `kind 23830` heartbeat events to the notifier when the user is online so that notifiers can avoid sending unnecessary notifications. Heartbeats should be one minute apart.
+
+In order to protect user privacy, the `content` should contain a [NIP-44](./44.md) encrypted JSON-encoded tag array with the following values:
+
+- One or more `a` tags indicate alert request addresses
+
+```jsonc
+{
+  "kind": 23830,
+  "tags": [
+    ["a", "32830:1238876738ce5bc6ddea7732e35e77f1216c000657b3cff5c7ff26a11fd145d2:1294742"],
+  ]
+}
+```
+
+# Unsubscribing
+
+When a user no longer wants to be notified, they may delete the alert by address, using a [NIP 09](./09.md) `kind 5` event with the `a` tag of the alert request. This event MUST be wrapped as described in [NIP 59](./59.md) and p-tagged to the alert provider so that providers don't need to maintain filters for a large number of `a` tags.
+
+# Alerts Status
+
+A provider SHOULD publish a `kind 32831` event which details the status of the user's alert. A non-existent alert status event indicates that no action has been taken. It MUST have the following tags, and no others:
+
+- A `d` tag set to the address of the `kind 32830` it refers to
+- A `p` tag indicating the author of the `kind 32830` alert event
+
+All other tags MUST be encrypted to the pubkey indicated by the `p` tag using NIP 44. The following tags are defined:
+
+- `status` SHOULD be one of `ok`, `pending`, `payment-required`, `error`
+- `message` SHOULD include human-readable information explaining the status of the alert
+
+# Discovery
+
+A provider MAY advertise its services via NIP 89 client listing.
```

Comment:
No. stop overloading NIP-89. its "Recommended Application Handlers" not "Advertise everything on nostr"

Reply:
You're right, I was advocating for the same idea over on #1942. I'll add a new kind.

Reply:
Changed this as well